gophernicus.h

```diff
@@ -274,7 +274,7 @@ size_t strlcat(char *dst, const char *src, size_t siz);
 /* Sizes & maximums */
 #define BUFSIZE		1024	/* Default size for string buffers */
 #define MAX_HIDDEN	32	/* Maximum number of hidden files */
-#define MAX_FILETYPES	128	/* Maximum number of suffix to filetype mappings */
+#define MAX_FILETYPES	1000	/* Maximum number of suffix to filetype mappings */
```

Comment:
The rest seem to keep to powers of two, can we keep that fashion here and maybe make it 1024? It just seems more uniform :)

Reply:
I'm tempted to make it 1023, just to keep people thinking... ^-^